CMakeLists.txt

```diff
@@ -1,29 +1,11 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 2.8.8)
```

Comment:
This upgrade can be easily avoided in case that's wished: It's only required to generate the `serialVersion.cmake` file (using `include(CMakePackageConfigHelpers)` and `write_basic_package_version_file()`), but the contents of that file is trivial and could simply be perssited